conformance/tests/generics_scoping.py

```diff
@@ -0,0 +1,80 @@
+# Specification: https://typing.readthedocs.io/en/latest/spec/generics.html#scoping-rules-for-type-variables
+
+# TODO: write PEP 695 versions
```

Comment:
These are already written (see [here](https://github.com/python/typing/blob/main/conformance/tests/generics_syntax_scoping.py)), so you can remove this comment.